[REF] addons_vauxoo: migrate controller_report_xls task#29976

goliveirab opened the pull request:
- migrate controller_report_xls migrate
- set product_pricelist_report_qweb

Commit:
[REF] addons_vauxoo: migrate controller_report_xls

- migrate controller_report_xls migrate
- set product_pricelist_report_qweb

Commit:
adding requerement

luisg123v reviewed (changes requested):
Hi @goliveirab,

You're not including the commit history, which:
- Makes the history to be lost
- Makes harder to review your actual changes

To migrate a module preserving the history, please follow [these instructions](https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-12.0#technical-method-to-migrate-a-module-from-110-to-120-branch)

Commit:
[FIX] make it installable

Commit:
[REF] controller_report_xls: complete migration to v11

- User ir.actions.report to generate html report to get xls
- remove controllers
- create reports

Commit:
[FIX] EOF

goliveirab commented:
Changed to https://github.com/Vauxoo/addons-vauxoo/pull/1466
